### DISCUSSION TITLE
Dot Net Mona Messages

### DISCUSSION THREAD

**Discussion by mishmanners** [mishmanners/GitHub-Like-A-Boss#2] [General] [answered]
Post all your awesome images of dotnetmona here.

Instructions:
1. go to: http://mishmanners.azurewebsites.net/
2. type at the end of the URL: "/?m= " and then your message
3. Post your pics

**Comment by martinwoodward** [comment]
![image](https://user-images.githubusercontent.com/856858/82414450-fbcfc880-9a6e-11ea-8b12-75a2de6b615d.png)
http://mishmanners.azurewebsites.net/?m=Show%20ma%20teh%20codez

**Reply by mishmanners** [reply]
I ❤️ this so much!

**Reply by ryanrichard19** [reply]
<img width="1215" alt="Screenshot 2020-05-20 at 08 53 34" src="https://user-images.githubusercontent.com/24666535/82414609-9af8be00-9a77-11ea-9f22-3fc91d49c710.png">

**Reply by martinwoodward** [reply]
![image](https://user-images.githubusercontent.com/856858/82414725-6a148b00-9a6f-11ea-87fa-f3a9ca380fa0.png)

**Reply by mishmanners** [reply]
![Tea](https://user-images.githubusercontent.com/36594527/82414780-ebddd600-9aba-11ea-9f94-552230483e83.PNG)

**Reply by martinwoodward** [reply]
```
        
         MMM.           .MMM                                                         (O)
         MMMMMMMMMMMMMMMMMMM                                                         ||
         MMMMMMMMMMMMMMMMMMM      _______________                             .DDDDDDDDDDDDDDDD.
        MMMMMMMMMMMMMMMMMMMMM    |               |                         .DDDDDDDDDDDDDDDDDDDDDD.
       MMMMMMMMMMMMMMMMMMMMMMM   | Tim Tam Slam? |                      .DDDDDDDDDDDDDDDDDDDDDDDDDDDD.
      MMMMMMMMMMMMMMMMMMMMMMMM   |_   ___________|                     .'DDDDDDDDDDDDDDDDDDDDdDD'  DDDD.
      MMMM::- -:::::::- -::MMMM    |/                                  :  .DDDDD.'.DDDDDDDDD         DDD.
       MM~:~ 00~:::::~ 00~:~MM                                         :    _            __        .DDDDD.
  .. MMMMM::.00:::+:::.00::MMMMM ..                                   .:    0            00        .DDDDD.
        .MM::::: ._. :::::MM.                                        .DDD       ^                 .DDDDD.
           MMMM;:::::;MMMM                                           .DDDD.  .DDDDD.          .DDDDDDDDDDD.
    -MM        MMMMMMM                                               .DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD'  .DDDDD.
    ^  M+     MMMMMMMMM                                           .DDDDDDDDDDD#NET#DDDDDDDDDDDdDDDDD'     .DDDDD.
        MMMMMMM MM MM MM                                       .DDDDDD.    .DdDDDDDDDDDDDDDDDDdDDDD.      .DDDDDDDD.
             MM MM MM MM                                    .DD'DDDDDD     DDDDDD. .DDDDDDDD....DD.         .DDDDD.
             MM MM MM MM                                  .dDDDDb       .DDDD'D.               .D'DDDD.
          .~~MM~MM~MM~MM~~.                                           .DDDDDD'D.               .D'DDDDDDD.
       ~~~~MM:~MM~~~MM~:MM~~~~                                     .DDDDDDDDD'D.               'D'DDDDDDDDD.
      ~~~~~~==~==~~~==~==~~~~~~                                  .DDDDDDDDDDD'D.               'D'DDDDDDDDDDD.
       ~~~~~~==~==~==~==~~~~~~                                  .DDDDDDDDDDDDD.                 D'DDDDDDDDDDD.
           :~==~==~==~==~~                                        .DDDDDD.
    
```

**Reply by iamrx0** [reply]
<img width="941" alt="Bildschirmfoto 2020-05-20 um 08 57 23" src="https://user-images.githubusercontent.com/22222279/82414859-f925a100-9a77-11ea-9206-1b9630a0339d.png">

**Comment by LEvinson2504** [comment]
Hey!

**Reply by mishmanners** [reply]
Hey @LEvinson2504

**Comment by bdougie** [comment]
![Screenshot 2020-05-19 23 54 25](https://user-images.githubusercontent.com/5713670/82414803-74616580-9a2c-11ea-9547-d2635c78b8bc.png)

**Comment by wrroper** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/2213614/82414844-8216eb00-9a2c-11ea-8fc2-ad858a6d742f.png)

**Reply by mishmanners** [reply]
![GithUbs](https://user-images.githubusercontent.com/36594527/82414938-28a9cd00-9abb-11ea-8e31-55aeec076b6b.PNG)

**Comment by johanbenschop** [comment]
![image](https://user-images.githubusercontent.com/5930361/82414945-1bb7ba00-9a78-11ea-9aab-ef4c12f052b4.png)

**Reply by mishmanners** [reply]
![Aussie](https://user-images.githubusercontent.com/36594527/82415052-4aa34f80-9abb-11ea-8d94-3f3deb4dc9b1.PNG)

**Comment by gsuttie** [comment]
Hi From Scotland!
<img width="686" alt="Gregor" src="https://user-images.githubusercontent.com/373279/82415068-e0b18880-9a6f-11ea-8e62-83764c65bff9.PNG">

**Reply by martinwoodward** [reply]
![wave,gif](https://user-images.githubusercontent.com/856858/82415111-eb6c1d80-9a6f-11ea-80c5-f291ec54ef0d.gif)

**Comment by smashinu** [comment]
![image](https://user-images.githubusercontent.com/30433218/82415299-e7112600-9aaa-11ea-9e46-a36679f3b530.png)

**Reply by mishmanners** [reply]
Yas!

**Comment by GoneFishern** [comment]
![Annotation 2020-05-20 170839](https://user-images.githubusercontent.com/197617/82415960-a4584980-9abc-11ea-99fa-848808d57018.png)

**Reply by mishmanners** [reply]
![Albury](https://user-images.githubusercontent.com/36594527/82419118-44b06d00-9ac1-11ea-9104-7fc6009af4db.PNG)

**Reply by GoneFishern** [reply]
lol. Awesome, I'm in Tallangatta. :-) just down the road.

**Reply by mishmanners** [reply]
Oh so close to me! I know where that is!!!

**Comment by lythix** [comment]
` // future self. Reminder to go and buy some more Github swag`

**Reply by lythix** [reply]
Oh look, I got bumped to the top. :)

**Comment by erwinheitzman** [comment]
```        
         MMM.           .MMM                                                         (O)
         MMMMMMMMMMMMMMMMMMM                                                         ||
         MMMMMMMMMMMMMMMMMMM      __________________________                  .DDDDDDDDDDDDDDDD.
        MMMMMMMMMMMMMMMMMMMMM    |                          |              .DDDDDDDDDDDDDDDDDDDDDD.
       MMMMMMMMMMMMMMMMMMMMMMM   |  Hi! Are thy Dr.Robotnik |           .DDDDDDDDDDDDDDDDDDDDDDDDDDDD.
      MMMMMMMMMMMMMMMMMMMMMMMM   |_   ______________________|          .'DDDDDDDDDDDDDDDDDDDDdDD'  DDDD.
      MMMM::- -:::::::- -::MMMM    |/                                  :  .DDDDD.'.DDDDDDDDD         DDD.
       MM~:~ 00~:::::~ 00~:~MM                                         :    _            __        .DDDDD.
  .. MMMMM::.00:::+:::.00::MMMMM ..                                   .:    0            00        .DDDDD.
        .MM::::: ._. :::::MM.                                        .DDD       ^                 .DDDDD.
           MMMM;:::::;MMMM                                           .DDDD.  .DDDDD.          .DDDDDDDDDDD.
    -MM        MMMMMMM                                               .DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD'  .DDDDD.
    ^  M+     MMMMMMMMM                                           .DDDDDDDDDDD#NET#DDDDDDDDDDDdDDDDD'     .DDDDD.
        MMMMMMM MM MM MM                                       .DDDDDD.    .DdDDDDDDDDDDDDDDDDdDDDD.      .DDDDDDDD.
             MM MM MM MM                                    .DD'DDDDDD     DDDDDD. .DDDDDDDD....DD.         .DDDDD.
             MM MM MM MM                                  .dDDDDb       .DDDD'D.               .D'DDDD.
          .~~MM~MM~MM~MM~~.                                           .DDDDDD'D.               .D'DDDDDDD.
       ~~~~MM:~MM~~~MM~:MM~~~~                                     .DDDDDDDDD'D.               'D'DDDDDDDDD.
      ~~~~~~==~==~~~==~==~~~~~~                                  .DDDDDDDDDDD'D.               'D'DDDDDDDDDDD.
       ~~~~~~==~==~==~==~~~~~~                                  .DDDDDDDDDDDDD.                 D'DDDDDDDDDDD.
           :~==~==~==~==~~                                        .DDDDDD.
    ```

**Comment by lythix** [comment]
I bumped the discussion. Very cool.

**Comment by erwinheitzman** [comment]
Of course we can also reference issues and other discussions from here #3 #1  \o/

**Comment by davibern** [comment]
![image](https://user-images.githubusercontent.com/13550820/82550185-6f043800-9b5e-11ea-82a8-08895c581576.png)

http://mishmanners.azurewebsites.net/?m=do%20you%20like%20pizza%20with%20pineapple?

**Reply by mishmanners** [reply]
![Pineapple](https://user-images.githubusercontent.com/36594527/82550890-95e45d80-9ba2-11ea-9fae-34448152a1ef.PNG)

Also loving the fact emojis work in ASCII art & C# 🍍

**Comment by manbirmarwah** [comment]
Oh God this discussion is pure GOLD! :joy::heart:

**Comment by Justinjdaniel** [comment]
I just discovered this discussion, and it is fantastic.
Since the URL is broken, I'm putting this as my message instead.

         MMM.           .MMM                                                         (O)
         MMMMMMMMMMMMMMMMMMM                                                         ||
         MMMMMMMMMMMMMMMMMMM      ___________________________                 .DDDDDDDDDDDDDDDD.
        MMMMMMMMMMMMMMMMMMMMM    |                           |              .DDDDDDDDDDDDDDDDDDDDDD.
       MMMMMMMMMMMMMMMMMMMMMMM   | Hey! Show me a dance move |           .DDDDDDDDDDDDDDDDDDDDDDDDDDDD.
      MMMMMMMMMMMMMMMMMMMMMMMM   |_   _______________________|          .'DDDDDDDDDDDDDDDDDDDDdDD'  DDDD.
      MMMM::- -:::::::- -::MMMM    |/                                  :  .DDDDD.'.DDDDDDDDD         DDD.
       MM~:~ 00~:::::~ 00~:~MM                                         :    _            __        .DDDDD.
     MMMMM::.00:::+:::.00::MMMMM ..                                   .:    0            00        .DDDDD.
        .MM::::: ._. :::::MM.                                        .DDD       ^                 .DDDDD.
           MMMM;:::::;MMMM                                           .DDDD.  .DDDDD.          .DDDDDDDDDDD.
    -MM        MMMMMMM                                               .DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD'  .DDDDD.
    ^  M+     MMMMMMMMM                                           .DDDDDDDDDDD#NET#DDDDDDDDDDDdDDDDD'     .DDDDD.
        MMMMMMM MM MM MM                                       .DDDDDD.    .DdDDDDDDDDDDDDDDDDdDDDD.      .DDDDDDDD.
             MM MM MM MM                                    .DD'DDDDDD     DDDDDD. .DDDDDDDD....DD.         .DDDDD.
             MM MM MM MM                                  .dDDDDb       .DDDD'D.               .D'DDDD.
          .~~MM~MM~MM~MM~~.                                           .DDDDDD'D.               .D'DDDDDDD.
       ~~~~MM:~MM~~~MM~:MM~~~~                                     .DDDDDDDDD'D.               'D'DDDDDDDDD.
      ~~~~~~==~==~~~==~==~~~~~~                                  .DDDDDDDDDDD'D.               'D'DDDDDDDDDDD.
       ~~~~~~==~==~==~==~~~~~~                                  .DDDDDDDDDDDDD.                 D'DDDDDDDDDDD.
           :~==~==~==~==~~                                        .DDDDDD.
